Subject: [PATCH] Make sdcard volumes visible even when FBE is enabled

Sdcards were no longer visible when FBE was enabled. This change makes
the sdcard visible so applications can still access and store content
on the card.
Issue introduced with commit 'Block adoptable storage when using FBE.'

diff --git a/services/core/java/com/android/server/MountService.java b/services/core/java/com/android/server/MountService.java
index 4cc2931..38f814f 100644
--- a/services/core/java/com/android/server/MountService.java
+++ b/services/core/java/com/android/server/MountService.java
@@ -1301,7 +1301,9 @@
 
             // Adoptable public disks are visible to apps, since they meet
             // public API requirement of being in a stable location.
-            if (vol.disk.isAdoptable()) {
+            // If FBE is enabled, sdcard is no longer considered adoptable,
+            // make sdcard visible.
+            if (vol.disk.isAdoptable() || vol.disk.isSd()) {
                 vol.mountFlags |= VolumeInfo.MOUNT_FLAG_VISIBLE;
             }
 
